### DISCUSSION TITLE
MIGX setup with standard input

### DISCUSSION THREAD

**Discussion by codexx20** [Bruno17/MIGX#412] [Q&A]
Hello,

Does anyone know if it is possible to create a MIGX TV with pre-filled values (and still possibility to add extra values like normal use). What I actually want to create is; 

MIGX TV actions_todolist with 5 input fields; 
todo_date (preferred execution date)
description (text field)
action_ready
action_ready date
notices (textarea)

The todo's that are standard for every new resource need to be prefilled, so whenever a new resource is created the standard todo's need to be created. Is there any possible way to set this up properly? 

Thanks in advance for replying to my question.

**Comment by Bruno17** [comment]
should be possible, to just put a json string with the default items into the default value of the MIGX TV
